tests/test_commands_logic.py

```
@@ -204,6 +204,89 @@ def download_chat_bytes(self, file_id):  # noqa: ARG002
+    result = sync_command(options, CommandEnv(ui=DummyUI()))
+
+    assert result.count == 1
+    expected_dir = out_dir / util.sanitize_filename("Drive Attachments")
+    assert (expected_dir / "conversation.md").exists()
+    assert (expected_dir / "attachments" / "good.txt").exists()
+    assert not (expected_dir / "attachments" / "bad.txt").exists()
+
+    assert int(result.total_stats.get("attachmentFailures", 0) or 0) == 1
+    failed_attachments = getattr(result, "failed_attachments", None)
+    assert isinstance(failed_attachments, list)
+    assert failed_attachments and failed_attachments[0]["id"] == "drive-1"
```

Ruff (0.15.9)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)